src/main/java/me/clip/placeholderapi/configuration/PlaceholderAPIConfig.java

```diff
@@ -63,4 +66,16 @@ public String booleanFalse() {
 	public String dateFormat() {
 		return plugin.getConfig().getString("date_format");
 	}	
+	
+	public String getTime(TimeFormat timeFormat) {
+		final FileConfiguration config = plugin.getConfig();
+		
+		switch (timeFormat) {
```

Comment:
This isn't a very extensible format to write code in.

Observing this code block, you will notice that the config path is directly reflected in the name of the associated enum constant. This entire block of code could be effectively one line.